website/content/docs/secrets/databases/mysql-maria.mdx

```diff
@@ -217,5 +217,16 @@ GRANT CREATEROLE ON <database>.<object> TO "test-user"@"%";
         service_account_json="@my_credentials.json"
     ```
 
-Once the connection has been configured and IAM authentication is complete, the steps to set up a role and generate
-credentials are the same as the ones listed above.
+1.  Configure a role in Vault as done above. Here you can override Vault's default revocation statements to simply
+    drop the user:
+
+    ```shell-session
+       $ vault write database/roles/my-role \
+           db_name=my-mysql-database \
+           creation_statements="CREATE USER '{{name}}'@'%' IDENTIFIED BY '{{password}}';GRANT SELECT ON *.* TO '{{name}}'@'%';" \
+           revocation_statements="DROP USER '{{name}}'@'%';" \
+           default_ttl="1h" \
+           max_ttl="24h"
+    ```
+
+Once the role configuration is complete, the steps generate credentials are the same as the ones listed above.
```

Comment:
We should list the actual commands here, not send the reader elsewhere in the doc. If we expect to use the steps under multiple situations, we can always make those steps a partial so it's easier to repeat them.